integration/client-cli/client-registration-cli/src/main/java/org/keycloak/client/registration/cli/commands/AbstractAuthOptionsCmd.java

```diff
@@ -151,6 +150,8 @@ protected void setupTruststore(ConfigData configData, CommandInvocation invocati
             } catch (Exception e) {
                 throw new RuntimeException("Failed to load truststore: " + truststore, e);
             }
+        } else {
+            HttpUtil.setSkipCertificateValidation();
```

Comment:
We shouldn't automatically skip certificate validation. Rather there should be an option to explicitly enable that if required.

Reply:
Granted and fixed.

I decided to completely change my strategy for the CLI tests. I reverted most of my changes and now use HTTP (and not HTTPS) port. Note, that the CLI tests explicitly test TLS/non-TLS scenario (for example in `KcAdmTruststoreTest#testTruststore`) so both, TLS and non-TLS scenarios are covered.